### PR TITLE
Create Definition_of_done.txt

### PR DESCRIPTION
**Pull request by gu92maak**
please all review the proposed DoD.

### CODE REVIEW

**Review by eliascalzone — DISMISSED** [review]

File: project_documentation/Definition_of_done.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Definition of Done:
+
+a User Story (US) is considered Done when all of the following are fullfilled:
+* The assigned reviewer(s) has checked that all the Acceptance Criterias are fullfilled, if there are any Acceptance Criterias
+* All the tasks related to the US are Done
+* The reviewer(s) has tested the US functionality in Android studio and not noticed any errors
+* if the US functionality as a whole was branched out, the Pull Request process has been completed
```
Comment (eliascalzone) [inline project_documentation/Definition_of_done.txt:7]: I don't quite understand this line, can you elaborate?

Comment (eliascalzone) [reply project_documentation/Definition_of_done.txt:7]: Line 7

Comment (antonsvaren) [reply project_documentation/Definition_of_done.txt:7]: Not sure I understand line 7

**Review by antonsvaren — CHANGES_REQUESTED** [review]
I think everything here correlates with what we have been talking about. I don't really understand Line7 though.

File: project_documentation/Definition_of_done.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* The reviewer(s) has tested the US functionality in Android studio and not noticed any errors
+* if the US functionality as a whole was branched out, the Pull Request process has been completed 
+
+Currently we have no coding guidelines that guide the reviewer on what to react on,
+Currently we have no requirements on level of unit test, regression tests or manual tests. only requirement on testing is that Aceptance Criteras shall be fullfilled and the US should be resolved.
+
+
+
+Pull Request process:
+When functionality is beeing beeing developed in a separate branch the branch shall be rebased, if main has evolved, and merged.
```
Comment (MaidScientistIzutsumiMarin) [inline project_documentation/Definition_of_done.txt]: "beeing beeing" is a little confusing

Comment (antonsvaren) [reply project_documentation/Definition_of_done.txt]: what do you mean? Its perfect 😃

**Review by Hannapham1007 — COMMENTED** [review]
DoD looks good, however I don't really get line 7

File: project_documentation/Definition_of_done.txt (hunk 1 above)
Comment (gu92maak) [reply project_documentation/Definition_of_done.txt:7]: "if the US functionality as a whole was branched out, the Pull Request process has been completed "
What I tried to say was:
If we choose to make a seperate branch for not just a task but instead for the whole User Story, 
then we should complete the Pull Request process (as it is defined on line 14 and onward), before we can tick this part of the Definition of Done.

Did I become more clear, @antonsvaren  and @eliascalzone ?

File: project_documentation/Definition_of_done.txt (hunk 2 above)
Comment (gu92maak) [reply project_documentation/Definition_of_done.txt]: thanks, I have updated in latest version.

now it says
"When functionality is being developed  ..."

is is this issue resolved, @ItsPonks ? if so please press resolve converssation for this issue.

**Review by Hannapham1007 — APPROVED** [review]

File: project_documentation/Definition_of_done.txt (hunk 1 above)
Comment (eliascalzone) [reply project_documentation/Definition_of_done.txt:7]: Yes, resolving.

**Review by eliascalzone — APPROVED** [review]

**Review by antonsvaren — APPROVED** [review]

**Review by irenecanovaas — APPROVED** [review]

**Review by alesanchez14 — APPROVED** [review]

**Pull request merged**